posthog/year_in_posthog/sharing-buttons.html

```diff
@@ -0,0 +1,34 @@
+<div>Share to</div>
+<div>
+    <a target="_blank"
+       aria-label="share year in posthog 2022 to facebook"
+       data-attr="share-year-in-posthog-2022-to-facebook"
+       href="http://www.facebook.com/sharer/sharer.php?u={{ page_url | urlencode }}&quote=No hedgehogs were harmed in the course of this year on @posthog">
+        <svg viewBox="0 0 24 24">
+            <path fill="white"
+                  d="M12 2.04C6.5 2.04 2 6.53 2 12.06C2 17.06 5.66 21.21 10.44 21.96V14.96H7.9V12.06H10.44V9.85C10.44 7.34 11.93 5.96 14.22 5.96C15.31 5.96 16.45 6.15 16.45 6.15V8.62H15.19C13.95 8.62 13.56 9.39 13.56 10.18V12.06H16.34L15.89 14.96H13.56V21.96A10 10 0 0 0 22 12.06C22 6.53 17.5 2.04 12 2.04Z"/>
+        </svg>
+    </a>
+</div>
+<div>
+    <a target="_blank"
+       aria-label="share year in posthog 2022 to linked in"
+       data-attr="share-year-in-posthog-2022-to-linked-in"
+       href="https://www.linkedin.com/sharing/share-offsite/?url={{ page_url | urlencode }}">
+        <svg viewBox="0 0 24 24">
+            <path fill="white"
+                  d="M19 3A2 2 0 0 1 21 5V19A2 2 0 0 1 19 21H5A2 2 0 0 1 3 19V5A2 2 0 0 1 5 3H19M18.5 18.5V13.2A3.26 3.26 0 0 0 15.24 9.94C14.39 9.94 13.4 10.46 12.92 11.24V10.13H10.13V18.5H12.92V13.57C12.92 12.8 13.54 12.17 14.31 12.17A1.4 1.4 0 0 1 15.71 13.57V18.5H18.5M6.88 8.56A1.68 1.68 0 0 0 8.56 6.88C8.56 5.95 7.81 5.19 6.88 5.19A1.69 1.69 0 0 0 5.19 6.88C5.19 7.81 5.95 8.56 6.88 8.56M8.27 18.5V10.13H5.5V18.5H8.27Z"/>
+        </svg>
+    </a>
+</div>
+<div>
+    <a target="_blank"
+       aria-label="share year in posthog 2022 to twitter"
+       data-attr="share-year-in-posthog-2022-to-twitter"
+       href="https://twitter.com/intent/tweet?text={{ 'No hedgehogs were harmed in the course of this year on @posthog' }}&url={{ page_url | urlencode }}&hashtags=YearInPostHog">
+        <svg viewBox="0 0 24 24">
```

Comment:
@elonmusk won't be happy

<img width="278" alt="Screenshot 2023-12-06 at 14 07 39" src="https://github.com/PostHog/posthog/assets/6685876/458d2f05-b493-4e41-bb87-309fe0a21959">

Reply:
I only just noticed you tagged him... now we're in trouble 🤣